administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -45,10 +60,16 @@ const BirthdayForm = ({
         onChange={changeBirthday}
         onClear={() => setValue({ birthday: null })}
         isValid={isValid || !showErrorMessage}
+        isHint={isBirthdayHintEnabled()}
         maxDate={new Date()}
         disableFuture
       />
-      {showErrorMessage && <FormErrorMessage errorMessage={getErrorMessage()} />}
+      {showErrorMessage && (
+        <FormErrorMessage
+          style={{ color: isBirthdayHintEnabled() ? 'black' : undefined }}
+          errorMessage={getErrorMessage()}
+        />
```

Comment:
:wrench:
i would add here a `showBirthdayHint` and separate hint and error handling

Reply:
I will add a prop called `isHint` becouse `FormErrorMessage` is used in multiple places